client/ayon_houdini/api/lib.py

```diff
@@ -1596,3 +1597,10 @@ def connect_file_parm_to_loader(file_parm: hou.Parm):
         f' {file_parm.node().path()} {file_parm.name()} \`{expression}\`'
     )
     show_node_parmeditor(load_node)
+
+
+def format_as_collections(files: list[str], pattern: str = "{head}{padding}{tail} [{ranges}]") -> list[str]:
```

Comment:
Let's add a small docstring
```suggestion
def format_as_collections(files: list[str], pattern: str = "{head}{padding}{tail} [{ranges}]") -> list[str]:
    """Return list of files as formatted sequence collections."""
```

Reply:
https://github.com/ynput/ayon-houdini/pull/193/commits/78e15a5ed327a27eabacc3dfa688d3d39c6e1557